src/frontend/src/btc/components/send/BtcReviewNetwork.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	import { BTC_REGTEST_NETWORK, BTC_TESTNET_NETWORK, BTC_MAINNET_NETWORK } from '$env/networks.env';
+	import ReviewNetwork from '$lib/components/send/ReviewNetwork.svelte';
+	import type { NetworkId } from '$lib/types/network';
+	import { isNetworkIdBTCTestnet, isNetworkIdBTCRegtest } from '$lib/utils/network.utils';
+
+	export let networkId: NetworkId | undefined = undefined;
+</script>
+
+<ReviewNetwork
+	sourceNetwork={isNetworkIdBTCTestnet(networkId)
```

Comment:
For another PR: it feels like this PR comes over and over again. Is there a way to refactor to avoid duplication of codes?

Reply:
yeah, I can do another mapper, similar to what has just been merged to main - https://github.com/dfinity/oisy-wallet/commit/8c6e765f7db196942e7385a16ce5cfa6328ad0fb

Reply:
If possible not "another" mapper, but just once.